packages/jss/tests/integration/plugins.js

```diff
@@ -406,27 +406,58 @@ describe('Integration: plugins', () => {
       `)
     })
 
-    it('should not call the hook if rule has no .style', () => {
```

Comment:
tests should also be in the plugin

Reply:
Yes. But I think it depends on the points I gave in the other comment :)